lib/layouts/conversation_list/pinned_conversation_tile.dart

```diff
@@ -0,0 +1,486 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:assorted_layout_widgets/assorted_layout_widgets.dart';
+import 'package:bluebubbles/blocs/chat_bloc.dart';
+import 'package:bluebubbles/helpers/socket_singletons.dart';
+import 'package:bluebubbles/helpers/utils.dart';
+import 'package:bluebubbles/layouts/conversation_list/pinned_tile_text_bubble.dart';
+import 'package:bluebubbles/layouts/conversation_view/conversation_view.dart';
+import 'package:bluebubbles/layouts/widgets/contact_avatar_group_widget.dart';
+import 'package:bluebubbles/layouts/widgets/message_widget/reactions_widget.dart';
+import 'package:bluebubbles/layouts/widgets/message_widget/typing_indicator.dart';
+import 'package:bluebubbles/layouts/widgets/theme_switcher/theme_switcher.dart';
+import 'package:bluebubbles/managers/current_chat.dart';
+import 'package:bluebubbles/managers/new_message_manager.dart';
+import 'package:bluebubbles/managers/settings_manager.dart';
+import 'package:bluebubbles/repository/models/chat.dart';
+import 'package:bluebubbles/repository/models/handle.dart';
+import 'package:bluebubbles/repository/models/message.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:get/get.dart';
+
+class PinnedConversationTile extends StatefulWidget {
+  final Chat chat;
+  final bool? onTapGoToChat;
```

Comment:
Can we make this non nullable? I'd prefer all bools to be strictly true/false

Reply:
I agree

Reply:
What is this variable for? It's not used anywhere. Neither is onTapCallback

Reply:
@jjoelj it's used in Material theme, when selecting multiple chats, I think?

Reply:
Nah @tneotia there's `inSelectMode` and `onSelect` for that. These have 0 code occurrences lol

Reply:
> Nah @zlshames there's `inSelectMode` and `onSelect` for that. These have 0 code occurrences lol

Well then i guess it's not used anymore. You can remove it

Reply:
@zlshames already done